pkg/dirs/dirs_test.go

```diff
@@ -8,43 +8,46 @@ import (
 	"testing"
 )
 
-func TestWindmillDir(t *testing.T) {
+func TestTiltDevDir(t *testing.T) {
 	emptyPath := ""
 	oldWmdaemonHome := os.Getenv("WMDAEMON_HOME")
 	oldHome := os.Getenv("HOME")
-	oldWindmillDir := os.Getenv("WINDMILL_DIR")
+	oldTiltDevDir := os.Getenv("WINDMILL_DIR")
```

Comment:
a. should this be oldWindmillDir?
b. do you have tests for how .windmill interacts with .tilt-dev? (not persuaded you need them tho)

Reply:
done, ya, there are tests for that